packages/terra-form-validation/src/terra-dev-site/doc/form-validation/MaxLengthInput.5.doc.mdx

```diff
@@ -0,0 +1,10 @@
+import { Badge } from 'terra-form-validation/package.json?dev-site-package';
+
+import TestFormValidation from '../example/MaxLengthInput?dev-site-example';
+
+<Badge />
+
+# Making an Input With a Maximum Length
+
+## Example
+<TestFormValidation title='Max Length Input' />
```

Comment:
We can move this example to the main [Form Validation Page](https://terra-framework-deploy-pr-1046.herokuapp.com/components/terra-form-validation/form-validation/form-validation) to avoid a separate Page just for one example.

Reply:
Changed e52f755